www/_posts/2023-12-01-survey.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author:
+    name: Niklas Merz
+    url: https://twitter.com/niklasmaerz
+title:  "Announcing the Cordova User & Contributor Survey!"
+categories: announcements
+tags: news
+---
+
+The Apache Cordova projects wants to understand the users and contributors better. Therefore, we invite you take part of a short survey for app developers and contributors of Cordova plugins.
+
+The survey questions are designed to not contain any personal information and can be answered anonymously. *Please take note that the survey is using Google Forms to process the responses. The [Google privacy policy](https://policies.google.com/privacy?hl=en) is applicable for the data processed.*
+
+**[Take survey](https://forms.gle/2HKWATLcRQdw85mE6)**
+
+Thank you very much for taking part in this survey. The goal of this survey is to identify areas that can be improved by the Cordova community together. The survey is open from now on to XXXXX. The results will be shared on the Cordova blog.
```

Comment:
I guess we should make sure we decide on a closing date ahead of time.

I'm not really sure how long is a long enough or too long... but I feel like it should be at least a month, but that means it's ending near the holidays so we might want to extend the end date until sometime in January perhaps.

Reply:
Agree. I suggested a timeline on the mailing list. What do you think?